rero_ils/modules/documents/serializers.py

```diff
@@ -114,14 +125,10 @@ def post_process_serialize_search(self, results, pid_fetcher):
                 metadata['ui_title_text_responsibility'] = text_title
 
             if viewcode != global_view_code:
-                items = metadata.get('items', [])
-                if items:
-                    output = []
-                    for item in items:
-                        if item.get('organisation')\
-                                .get('organisation_pid') == view_id:
-                            output.append(item)
-                    record['metadata']['items'] = output
+                record['metadata']['items'] = [
```

Comment:
You can maybe use a `filter()` function.

Reply:
I tried :-) But using a `filter` is less efficient than a comprehension list and the line length of the lambda function always are more than 80 characters) :D